CRM/Report/Form/Member/Detail.php

```diff
@@ -27,7 +27,7 @@ class CRM_Report_Form_Member_Detail extends CRM_Report_Form {
     'Organization',
   ];
 
-  protected $_customGroupGroupBy = FALSE;
+  protected $_customGroupGroupBy;
```

Comment:
It seems wrong to change this but I don't see anywhere where it ends up making a difference.